python_modules/libraries/dagster-celery-docker/dagster_celery_docker/executor.py

```diff
@@ -272,15 +281,32 @@ def _execute_step_docker(
         if docker_config.get("env_vars"):
             docker_env = {env_name: os.getenv(env_name) for env_name in docker_config["env_vars"]}
 
+        container_kwargs = docker_config.get("container_kwargs")
+        container_kwargs = check.opt_dict_param(container_kwargs, "container_kwargs", key_type=str)
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"startLine":285,"endLine":285,"prLink":"https://app.graphite.dev/github/pr/dagster-io/dagster/7335"}} -->
<i class='graphite__hidden'>[Re: lines 285 to 285]<br/></i>

can combine these lines<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/7335?utm_source=unchanged-line-comment">Graphite</a>.</span>